universal-application-tool-0.0.1/app/repository/UserRepository.java

```diff
@@ -102,10 +106,23 @@ public UserRepository(
         () -> {
           Optional<Account> accountMaybe = lookupAccount(emailAddress);
           // Return the applicant which was most recently created.
-          return accountMaybe.flatMap(
-              account ->
-                  account.getApplicants().stream()
-                      .max(Comparator.comparing(compared -> compared.getWhenCreated())));
+          // If no applicant exists, this is probably an account waiting for
+          // a trusted intermediary - create one.
```

Comment:
No, it means create one here - this method does that on line 122.  I'm not planning to modify the comment since I think that's the natural reading of it - if you have a suggestion I'd take it!